react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -222,7 +230,25 @@ private void updateSelectionIfNeeded(ReactAztecText view, @Nullable ReadableMap
         if ( selection != null ) {
             int start = selection.getInt("start");
             int end = selection.getInt("end");
-            view.setSelection(start, end);
+            int textLength = view.getText().length();
+            boolean startAndEndAreValid = start >= 0 &&
+                                          end >= 0 &&
+                                          start <= textLength &&
+                                          end <= textLength;
+            if (startAndEndAreValid) {
+                view.setSelection(start, end);
+            } else if (exceptionLogger != null) {
+                // Calling view.setSelection would have thrown an exception, so let's send information about
```

Comment:
Kinda a nit but, wdyt about instantiating and `printStackTrace`ing the exception anyway, and only guard the `exceptionLogger.accept(exception)` with the exceptionLogger check (`if (exceptionLogger != null)`)? That way we can still log locally that we swallowed the expeption even if a logger is not provided.

Or is it that you completely want to avoid logging it if a logger is not provided?

Reply:
> Or is it that you completely want to avoid logging it if a logger is not provided?

Nope, was just an oversight on my part. Updated to always print the stacktrace without regard to whether there is an exception logger in 0f3483d.